docs/translation/README.md

```diff
@@ -0,0 +1,91 @@
+
+You can either download a raw file from GitHub (by clicking on the desired file and clicking on the “download” button) OR you can use a web development IDE to work on the whole Strype project directly (cloning the project from GitHub with GIT). This option requires more preparation work (see the project’s <a href="../../README.md" target="_blank">README</a> for more details) **but will allow you to test your translation locally**.
+
+If the locale you wish to provide translation for does *not* exist, you will need to create a new subfolder (named after the 2-letter code of the locale) and add your translation files following the same naming logic than mentioned above. The easiest way is to copy one of the `en_*.json`, rename it with the right locale code, and edit it with your translations.
+
+>[!NOTE]
+>It is advised, if you want to retrieve a copy of the Strype source code, to first create a fork of the project into your own GitHub account, and use your forked project as a remote GIT repository.
+
+## How to understand the translations files
+
+The translation files are <a href="https://en.wikipedia.org/wiki/JSON" target="_blank">JSON</a> files: formatted text files that define **_key_:_value_ pairs**. There are several files per language, identified by the corresponding locale and gathered inside one subfolder for that locale (see section [1](#how-to-retrieve-or-create-the-translation-files) for more details).
+
+The *key* is used by Strype as *a reference* to a localisable text, independent of the locale.  
+The corresponding *value* is the localised text *that will be displayed* in Strype, in a given locale.  
+The *key* and its corresponding *value* are separated by a colon `:`.
+
+Here is an example of a *key:value* pair for English, where the *key* (i.e.reference) used by Strype is <img src="./img/bluecode-strypeFileDesc.png" alt="strypeFileDesc" align="center" width="136"> and the *value* displayed in Strype is <img src="./img/redcode-Strype project file.png" alt="text"  align="center" width="179">.
+
+<img src="./img/translations_KVP_example.png">
+
+>[!CAUTION]
+>Only the **text** values need to be translated.</span>
+
+## How to do the translations
+
+The language reference for all translations is the English locale. Therefore, the reference files are the files located in the `/en/` subfolder (see sections [1](#how-to-retrieve-or-create-the-translation-files) and [2](#how-to-understand-the-translations-files) for details about finding the files and understanding their formats).
+
+**Each** *key:value* pair of a translation locale file **need to match those of the equivalent English locale file**, and as explained above, **only** the text *values* need to be translated.
+
+Here are a few rules to follow when doing the translations of the text *values*.
```

Comment:
Probably worth mentioning to save the files in UTF-8 (but can point out this is the default for most modern text editors).